sources/web/datalab/static/datalab.js

```diff
@@ -277,6 +277,166 @@ function initializePage(dialog, saveFn) {
   });
 }
 
+// constants for minitoolbar operations
+const CELL_METADATA_COLLAPSED = 'hiddenCell';
+const CELL_METADATA_OUTPUT_COLLAPSED = 'hiddenOutput';
```

Comment:
Why is this not just the existing Jupyter [cell metadata of "collapsed"](https://nbformat.readthedocs.io/en/latest/format_description.html#metadata)?

I would prefer to not define our own metadata key if there is an existing one that does the same thing.